### DISCUSSION TITLE
rehype/mathjax

### DISCUSSION THREAD

**Discussion by JoaoVCMiranda** [nuxt/nuxt#28516] [Questions]
Hi there folks who are developing with Nuxt!

So, I've decided to migrate my blog to Nuxt, but I'm facing an issue when it comes to insert math equations.

From a few issues I've read on github, I came across this plugin called remarkMath and I can install a plugin(rehype/mathjax) in it to enable the rendering on math equations.

In the file `nuxt.config.ts`

```ts
export default defineNuxtConfig({
...
mdc:{
    remarkPlugins: {
      remarkMath: {src: 'remark-math'}},
	rehypePlugins: {
    rehypeMathjax: {
      src: 'rehype-mathjax',
    }
  }
```

Anyway, after researching for a few hours I still have no clue of **how can I customize** `rehype/mathjax` **elements**, aka. `mjx-container`
to make them relative to my `<p>` tags.

How it looks now:
![rehype-mathjax](https://github.com/user-attachments/assets/d902eb0e-f678-4bca-9b22-249e0ab97cd5)
How it was with jekyll
![jekyll](https://github.com/user-attachments/assets/c93ec216-e737-4664-86b3-ac01ab663221)

And the block equations are also not exactly perfect yet because I couldn't attribute the current color to it.

Thanks in advance!
PS: this interesting warning also is been shown several times on the server console:
```
 WARN  [Vue warn]: Failed to resolve component: G
If this is a native custom element, make sure to exclude it from component resolution via compilerOptions.isCustomElement.


 WARN  [Vue warn]: Failed to resolve component: Use
If this is a native custom element, make sure to exclude it from component resolution via compilerOptions.isCustomElement.


 WARN  [Vue warn]: Failed to resolve component: Rect
If this is a native custom element, make sure to exclude it from component resolution via compilerOptions.isCustomElement.


 WARN  [Vue warn]: Failed to resolve component: MjxContainer
If this is a native custom element, make sure to exclude it from component resolution via compilerOptions.isCustomElement.


 WARN  [Vue warn]: Failed to resolve component: Defs
If this is a native custom element, make sure to exclude it from component resolution via compilerOptions.isCustomElement.


 WARN  [Vue warn]: Failed to resolve component: Path
If this is a native custom element, make sure to exclude it from component resolution via compilerOptions.isCustomElement.
```

**Comment by data-miner00** [comment]
I believe you can fix this issue with some CSS manipulation. For me, I am having another issue where the Svg tags generated have incorrect casing. For example, `Defs` should be `defs` and `Path` should be `path` instead. No idea how to resolve this at the moment 🤷‍♂️
